cfpb/consumerfinance.gov
# Remove related metadata half-width modifier

`m-related-metadata__half-width` was only used on two pages 

https://www.consumerfinance.gov/enforcement/investigatory-authority/petitions-to-enforce/educational-credit-management-corporation/

https://www.consumerfinance.gov/enforcement/investigatory-authority/petitions-to-modify-or-set-aside/fastbucks-holding-corporation/

This places the related metadata into columns at tablet size (only). This is an option that can be set in Wagtail. This PR removes that option. In the future the related metadata should be rewritten with flexbox to flow into columns automatically when there is room.

@csebianlander reports unchecking this option from the two pages listed. This PR will remove it entirely as an option.

## Removals

- Remove related metadata half-width modifier

## How to test this PR

1. Visit localhost:8000/enforcement/investigatory-authority/petitions-to-enforce/educational-credit-management-corporation/ and edit the page in wagtail and see that "Is half width" is not an option in the sidefoot's related metadata.
